contributors/release-note-checker.md

```diff
@@ -0,0 +1,17 @@
+## Background
+As we all know, the release process of TiDB is relatively lengthy. One of the most cost time thing is the collection of release notes. We have many PRs that do not have a release note when they are merged. Therefore, a release note checker was added to some repo in CI Check the release note, please pay attention when you mention pr and review, the effect is as follows
```

Comment:
```suggestion
As we all know, the release process of TiDB is relatively lengthy. One of the most time-consuming thing is collecting release notes. We have many PRs that do not have a release note when they are merged. Therefore, a release note checker has been added to some repositories in the CI Check. Please pay attention when you mention or review a PR. The effect is as follows:
```

Reply:
- I fail to understand why we need to pay attention when "mention" a PR?
- Do we need to tell the name of those repos with this checker?

Reply:
1. "mention" here means when you create a pr, I will fix it
2. I think not list the repos. I will comment on each PR which fail for checker to take a look at the document